.github/workflows/typos.toml

```diff
@@ -0,0 +1,32 @@
+[files]
+extend-exclude = ["*.eps", "*.css", "*.map", "*.svg"]
+
+[default.extend-words]
+# Ignore IDAES
+IDAES = "IDAES"
+idaes = "idaes"
+IDAE = "IDAE"
+# HELMET - HELMholtz
+HEL = "HEL"
+# Datas
+Datas = "Datas"
+# ba triggers in docs
+ba = "ba"
+# Ignore complimentarity
+complimentarity = "complimentarity"
+# Attemp == Attemperator
+Attemp = "Attemp"
+attemp = "attemp"
+# Ficks Law
+Ficks = "Ficks"
+# scaling - ue
+ue = "ue"
+# Real word - opulate
+opulate = "opulate"
```

Comment:
Do we actually have an instance of this - my suspicion is that this would be a mistyping of populate.

Reply:
We do! And I looked up the word to make sure it was real (it is) and it was used correctly in the context. Hence the addition here.